public/achievementInfo.php

```diff
@@ -121,8 +121,21 @@ function generateAchievementMetaDescription(
 RenderContentStart($pageTitle);
 ?>
 <?php if ($permissions >= Permissions::Developer || ($permissions >= Permissions::JuniorDeveloper && $isAuthor)): ?>
+    <?php
+        $canHaveTypes = mb_strpos($gameTitle, "[Subset") === false && mb_strpos($gameTitle, "~Test Kit~") === false;
+    ?>
+
     <script>
     function updateAchievementDetails() {
+        const canHaveTypes = <?= $canHaveTypes ? 'true' : 'false' ?>;
+
+        const typeInputValue = $('#typeinput').val();
+
+        if (!canHaveTypes && typeInputValue) {
+            showStatusFailure('Error: Types are not supported on subsets or test kits.');
```

Comment:
Recommend hiding the dropdown if the type is already None

Reply:
Removed this new JS and set the dropdown to never appear if `$canHaveTypes` is falsy. Related SQL query we can execute after shipping: https://discord.com/channels/476211979464343552/1174126386349342901